src/libcrun/seccomp.c

```diff
@@ -34,6 +34,8 @@
 #include <sys/sysmacros.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <stdatomic.h>
```

Comment:
This seems to need `__STDC_VERSION__ >= 201112L`

Reply:
And the file is missing in gcc 4.8 (CentOS 7)
https://stackoverflow.com/questions/20326604/stdatomic-h-in-gcc-4-8